### ISSUE TITLE
test.txt in train val split protocol

### ISSUE THREAD

**Issue by licj15** [qianguih/voxelnet#29]
Hi friends,
I am trying to split the training set into training and validation set according to the protocol. But I find the test.txt in it. But now we only have training and validation set from the KITTI official website. So what the test.txt stands for? Maybe it has some connection with the process of KITTI submission? A test dataset will give us with the test.txt if we want to submit our results on KITTI?

Thank you!

**Comment by johleh** [comment]
yes test.txt lists the 7517 frames of the testdata to submit results to the online benchmark.
train.txt and val.txt is all you need unless you want to submit your results.

**Comment by licj15** [comment]
Thank you! This helps me a lot.
